hybridesmbench/eval/_diags/sanity_checks_mean/data/clivi/metadata.yml

```
@@ -0,0 +1,40 @@
+? /home/b/b309059/HybridESMBench/hybridesmbench/eval/_diags/sanity_checks_mean/data/clivi/MultiModelMax_Amon_clivi_1982-2018.nc
```

```suggestion
? clivi/MultiModelMax_Amon_clivi_1982-2018.nc
```
Needs to be relative paths